utils/codepointtrie/src/codepointtrie.rs

```diff
@@ -104,14 +105,16 @@ fn maybe_filter_value<T: TrieValue>(value: T, trie_null_value: T, null_value: T)
 /// For more information:
 /// - [ICU Site design doc](http://site.icu-project.org/design/struct/utrie)
 /// - [ICU User Guide section on Properties lookup](https://unicode-org.github.io/icu/userguide/strings/properties.html#lookup)
-#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+// serde impls in crate::serde
 #[derive(Debug, Eq, PartialEq, Yokeable, ZeroFrom)]
 pub struct CodePointTrie<'trie, T: TrieValue> {
-    header: CodePointTrieHeader,
-    #[cfg_attr(feature = "serde", serde(borrow))]
-    index: ZeroVec<'trie, u16>,
-    #[cfg_attr(feature = "serde", serde(borrow))]
-    data: ZeroVec<'trie, T>,
+    pub(crate) header: CodePointTrieHeader,
+    pub(crate) index: ZeroVec<'trie, u16>,
+    pub(crate) data: ZeroVec<'trie, T>,
+    // serde impl skips this field
+    #[zerofrom(clone)] // TrieValue is Copy, this allows us to avoid
+    // a T: ZeroFrom bound
+    pub(crate) error_value: T,
```

Comment:
Suggestion: could we make a wrapper around `T` that implements `ZeroFrom`, something like

```rust
#[repr(transparent)]
pub struct CopyWrap<T: Copy>(pub T);

impl<T: Copy> ZeroFrom for CopyWrap<T> { ... }
```

and export that wrapper from the ZeroFrom crate?

Given that we don't have specialization in Rust, wrapper structs like this seem like the best path forward in these types of situations.

Reply:
Eh, this is kinda what #[zerofrom(clone)] is for. We could introduce #[zerofrom(copy)] if we really want to be sure, but clone works fine here, it's a Copy type.

Reply:
`#[zerofrom(clone)]` has code smell. I like `#[zerofrom(copy)]`, or my `CopyWrap` suggestion.

Reply:
Yeah but we can introduce that any time.